Gemfile.lock

```diff
@@ -0,0 +1,63 @@
+GEM
```

Comment:
Not sure if all here is pertinent (or even the file!) @MattiSG please review ;)

Reply:
Mmh. Actually this is very useful if we want a stable environment for CI. Currently, honestly, without CI, all the Gemfile stuff is useless as you could just `gem install jekyll && jekyll serve` locally without caring that much about the versions.

I quickly copied the instructions from another website in which I invested heavily on documentation and stability, without being careful enough about iterating over complexity.

I think for the moment the simplest thing to do is to remove all the `Gemfile`  variants and update the instructions to use `gem install jekyll -v` and specify the version currently used in production by GitHub Pages. This will yield inconsistencies at some point when GitHub Pages updates their own production version, we'll catch the error and bring in CI and a lock file to prevent it from happening again 😉